_api-reference/index-apis/rollover.md

```diff
@@ -0,0 +1,196 @@
+`timeout` | Time | How long to wait for a response. Default is `30s`.
+`wait_for_active_shards` | String | The number of active shards that must be available before OpenSearch processes the request. Default is `1` (only the primary shard). You can also set to `all` or a positive integer. Values greater than 1 require replicas. For example, if you specify a value of 3, the index must have two replicas distributed across two additional nodes for the operation to succeed.
+
+## Request body
+
+The following request body options are supported.
+
+### `alias`
+
+The alias name as the key. Required when the `template` option exists in the request body.
+
+The object body contains the following optional options for the alias:
+
+Parameter | Type | Description
+:--- | :--- | :---
+`filter` | Query DSL object | The query that limits documents the alias can access.
+`index_routing` | String | The value which routes indexing operations to a specific shard. When specified, overwrites the `routing` value for indexing operations.
+`is_hidden` | Boolean | When `true`, the alias is hidden. Default is false. All indexes for the alias must have matching values for this setting.
+`is_write_index` | Boolean | When `true`, the index is the [write index] for the alias. Default is `false`.
+`routing` | String | The value used to route index and search operations to a specific shard.
+`search_routing` | String | The value used to write specifically search operations to a specific shard. When specified, this option overwrites the `routing` value for search operations.
+
+### `mappings`
+
+The field mappings that exist in the index. For more information, see [Mappings and field types](https://opensearch.org/docs/latest/field-types/). Optional.
+
+### `conditions`
+
+The conditions parameter is an optional object that allows you to define specific criteria for triggering the rollover operation. When provided, OpenSearch only runs the rollover if the current index satisfies one or more of the specified conditions. However, if this parameter is omitted, OpenSerch will perform the rollover unconditionally, without any prerequisite conditions.
+
+The object body supports the following options.
+
+Parameter | Type | Description 
+:--- | :--- | :--- 
+| `max_age` | Time units | Triggers a rollover after the maximum elapsed time from index creation is reached. The elapsed time is always calculated since the index creation time, even if the index origination date is configured to a custom date, such as when using the `index.lifecycle.parse_origination_date` or `index.lifecycle.origination_date` settings. Optional. |
+`max_docs` | Integer | Triggers a rollover after the specified maximum number of documents is reached. Documents added since the last refresh are not included in the document count. The document count does not include documents in replica shards. Optional 
```

Comment:
```suggestion
`max_docs` | Integer | Triggers a rollover after the specified maximum number of documents, excluding documents added since the last refresh and documents in replica shards. Optional 
```

Reply:
+1, but add "is reached" after "maximum number of documents" and a period after "Optional."